packages/b/bulmajs.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "bulmajs",
+    "filename": "bulma.js",
+    "license": "MIT",
+    "description": "Unofficial javascript extension to the awesome Bulma CSS framework.",
+    "homepage": "https://bulmajs.tomerbe.co.uk",
+    "keywords": [
+        "javascript",
+        "bulma"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/VizuaaLOG/BulmaJS.git"
+    },
+    "autoupdate": {
+        "source": "npm",
+        "target": "@vizuaalog/bulmajs",
+        "fileMap": [
+            {
+                "basePath": "dist",
+                "files": [
+                    "**/*.+(js|map)"
```

```suggestion
                    "*.js?(.map)"
```
Recursive wildcards are dangerous, and we don't need it here.

(There are no map files, is `?(.map)` needed, are you planning to start publishing them?)